### ISSUE TITLE
Id and data options

### ISSUE THREAD

**Issue by pineapple-thief** [jamesmartin/inline_svg#7]
How about adding id and data options for this wonderful gem - much like original image_tag? It could be useful to have that extra flexibility. I really want to inline my svgs to reduce number of requests per page, but I want to define some data attributes on some of them.

**Comment by jamesmartin** [comment]
Good ideas. Let's stick to the Rails `image_tag` API as closely as possible. E.g.

```ruby
# Id attribute:
inline_svg("some.svg", id: "some-id")
# => <svg id="some-id">Some document</svg>

# Data attributes
inline_svg("some.svg", data: { foo: "bar", baz: "quux"})
# => <svg data-foo="bar", data-baz="quux">Some document</svg>
```

Is that what you would expect?

**Comment by pineapple-thief** [comment]
Yes, definitely stick to Rails API! That would be great.

**Comment by jamesmartin** [comment]
Fixed in [v0.5.0](https://github.com/jamesmartin/inline_svg/blob/master/CHANGELOG.md#050---2015-03-29).
